sdk/src/types/block/protocol.rs

```diff
@@ -180,46 +184,46 @@ impl ProtocolParameters {
 #[getset(get_copy = "pub")]
 pub struct WorkScoreStructure {
     /// Modifier for network traffic per byte.
-    data_kilobyte: u32,
+    work_score_data_kilobyte: u32,
```

Comment:
I think this is going to change to `kibibyte`. Also I asked whether we should drop the `work_score` prefix in the TIP.

Reply:
kibiwhatnow

Reply:
It's been changed